arbnode/batch_poster.go

```diff
@@ -1958,6 +2057,12 @@ func (b *BatchPoster) MaybePostSequencerBatch(ctx context.Context) (bool, error)
 		log.Info("Got successful receipt from batch poster transaction", "txHash", tx.Hash(), "blockNumber", receipt.BlockNumber, "blockHash", receipt.BlockHash)
 	}
 
+	// After successful batch post, reset useEthDA flag
+	if b.useEthDA {
+		log.Info("Successful EthDA post after AltDA failure. Will try AltDA next time.")
+		b.useEthDA = false
```

Comment:
Reverting back to normal DA after every batch is too aggressive.
I think simplest option is to have a config that has number of batches to post to ethDA before trying again. I think default should be no less than 10. Perhaps even maxint (they can manually go back to altDA with a simple restart). They can configure it to 1 if they want to.